dwds/pubspec.yaml

```
@@ -15,7 +15,7 @@ dependencies:
   collection: ^1.15.0
   crypto: ^3.0.2
   dds: ^2.7.1
-  file: ^7.0.0
+  file: ">=6.0.0 <8.0.0"
```

To be conservative, I'd recommend

```suggestion
  file: ">=7.0.0 <8.0.0"
```

---

OH wait, actually, this should be `>=7.0.0 <9.0.0`, right?

---

nvm, this is correct as is :)